buttonfun/buttonfun/ButtonFunCollectionViewCell.m

```diff
@@ -0,0 +1,35 @@
+//
+//  ButtonFunCollectionViewCell.m
+//  buttonfun
+//
+//  Created by Shannon Sullivan on 7/30/15.
+//  Copyright (c) 2015 Sam Courtney. All rights reserved.
+//
+
+#import "ButtonFunCollectionViewCell.h"
+
+NSString *const BUTTON_FUN_CELL_REUSE_IDENTIFIER = @"ButtonFunCollectionViewCellIdent";
+
+@implementation ButtonFunCollectionViewCell
+
+- (instancetype)initWithFrame:(CGRect)frame {
+    // This is called when we are first created - make sure we start off with a random color
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self setRandomColor];
+    }
+    
+    return self;
+}
+
+- (void)setRandomColor {
```

Comment:
I think this should probably be broken out into  a category on UIColor for re-usability

Reply:
![lasersam](https://user-images.githubusercontent.com/1574781/30877314-52b22d20-a2c7-11e7-8a3d-9b50a2662162.png)